ci-operator/step-registry/hypershift/mce/agent/create/hypershift-mce-agent-create-chain.metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"path": "hypershift/mce/agent/create/hypershift-mce-agent-create-chain.yaml",
+	"owners": {
+		"approvers": [
+			"csrwng",
```

Comment:
@enxebre should we (@bryan-cox and me) to be included in the Approvers/Reviewers?